components/automate-ui/src/app/entities/rules/rule.reducer.ts

```diff
@@ -30,7 +74,49 @@ export const RuleEntityInitialState: RuleEntityState = ruleEntityAdapter.getInit
   createStatus: EntityStatus.notLoaded,
   createError: null,
   deleteStatus: EntityStatus.notLoaded,
-  updateStatus: EntityStatus.notLoaded
+  updateStatus: EntityStatus.notLoaded,
+  getAttributes: {
+    node: [
```

Comment:
Please use the constant `Attributes` that you just defined above instead of duplicating the list here.